.github/workflows/ci.yml

```
@@ -242,8 +242,1000 @@ jobs:
       matrix:
```

This is normally how I do it. (Note that it needs to be on the same level as `include`.

```suggestion
      matrix:
      	dim1: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
      	dim2: ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j']
```

---

Hmm that seems to skip the `direct` test. Even copying them brute force seems to cause failures. Wondering now if it's something screwy with GH actions. Very strange

---

Ah, need to fix main artifact name